README.md

```diff
@@ -363,6 +363,30 @@ const options = {
 }
 ```
 
+## Collaboration
+Thanks for taking an interest in the library and the github community!
+
+The following commands should get you started:
+
+Fork the project and clone the project in a local repository
+Add a new brunch
```

Comment:
We can probably remove this line and the extra line break below. I think we can leave out specific branch and naming conventions for now, as forking is good enough to get started.